src/parsing.jl

```diff
@@ -67,6 +67,55 @@ function composed_or_symbol(x::Expr)
         all(composed_or_symbol, x.args[2:end])
 end
 
+is_call(x) = false
+is_call(x::Expr) = x.head == :call
+
+is_nested_fun(x) = false
+function is_nested_fun(x::Expr)
+    x.head == :call &&
+    length(x.args) == 2 &&
+    is_call(x.args[2]) &&
+    # AsTable(:x) or `$(:x)`
+    get_column_expr(x.args[2]) == nothing
+end
+
+is_nested_fun_recursive(x, nested_once) = false
+function is_nested_fun_recursive(x::Expr, nested_once)
```

Comment:
Here and below are two recursive functions. `is_nested_fun_recursive` simply checks whether or not an expression is of the form `f(g(:x, :y))` and `make_composed` actually creates `(f ∘g)` . It may be too costly to do this recursive search twice but both should return relatively quickly.

Reply:
I think performance should not be crucial here.